src/web/app/pages-instructor/instructor-student-records-page/instructor-student-records-page.component.html

```diff
@@ -6,7 +6,7 @@ <h2>Records in feedback sessions</h2>
       <div class="card-header cursor-pointer" (click)="session.isCollapsed = !session.isCollapsed">
         <strong class="text-break">Feedback Session: {{ session.feedbackSession.feedbackSessionName }}</strong>
         <div class="card-header-btn-toolbar">
-          <tm-panel-chevron [isExpanded]="!session.isCollapsed"></tm-panel-chevron>
+          <tm-panel-chevron [isExpanded]="!session.isCollapsed" [chevronColor]="'black'"></tm-panel-chevron>
```

Comment:
I think let's just go with `chevronColor="black"` without the [] property binding, since we are just assigning a static value (this is also aligned with the `instructor-session-result-question-view` component)